app/models/User.coffee

```diff
@@ -513,6 +513,7 @@ module.exports = class User extends CocoModel
   hideTopRightNav: -> @isTarena()
   hideFooter: -> @isTarena()
   useGoogleClassroom: -> not (features?.chinaUx ? false) and me.get('gplusID')?   # if signed in using google SSO
+  useGoogleAnalytics: -> not (features?.chinaInfra ? false)
```

Comment:
Using google analytics seems like more of a application level decision (based on infra), not quite a user level decision.

Do we have the concept of global / application level tags?  May not be appropriate for this PR but want to call out for discussion / my own knowledge.

Reply:
We are currently overloading the User model with features as it's globally accessible.
The various tiers of "features" are flattened in the server and then delivered to the client under the global namespace `features`. This can make it confusing. For example. `features.chinaUx` is a global feature flag that can share the `features` object with a user local feature flag.